src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -2971,14 +2971,16 @@ namespace PackagedTerminalHeatPump {
 			MyPlantScanFlag( PTUnitNum ) = false;
 		}
 
-		if ( ! ZoneEquipmentListChecked && ZoneEquipInputsFilled ) {
-			ZoneEquipmentListChecked = true;
-			for ( Loop = 1; Loop <= NumPTUs; ++Loop ) {
-				if ( CheckZoneEquipmentList( PTUnit( Loop ).UnitType, PTUnit( Loop ).Name, CtrlZoneNum ) ) {
-					// save the ZoneEquipConfig index for this unit
-					PTUnit( Loop ).CtrlZoneNum = CtrlZoneNum;
-				} else {
-					ShowSevereError( "InitPTHP: Packaged Terminal Unit=[" + PTUnit( Loop ).UnitType + ',' + PTUnit( Loop ).Name + "] is not on any ZoneHVAC:EquipmentList.  It will not be simulated." );
+		if ( ZoneEquipmentListNotChecked ) {
+			if( ZoneEquipInputsFilled ) {
+				ZoneEquipmentListNotChecked = false;
+				for ( Loop = 1; Loop <= NumPTUs; ++Loop ) {
+					if ( CheckZoneEquipmentList( PTUnit( Loop ).UnitType, PTUnit( Loop ).Name, CtrlZoneNum ) ) {
+						// save the ZoneEquipConfig index for this unit
+						PTUnit( Loop ).CtrlZoneNum = CtrlZoneNum;
+					} else {
+						ShowSevereError( "InitPTHP: Packaged Terminal Unit=[" + PTUnit( Loop ).UnitType + ',' + PTUnit( Loop ).Name + "] is not on any ZoneHVAC:EquipmentList.  It will not be simulated." );
+					}
```

Comment:
This one-time check was executing every time through Init. Changed logical to static and re-organized to be more efficient.